main/src/cgeo/geocaching/CacheDetailActivity.java

```diff
@@ -2353,11 +2354,9 @@ public void call() {
     }
 
     public static void editPersonalNote(final Geocache cache, final CacheDetailActivity activity) {
-        if (cache.isOffline()) {
```

Comment:
Not sure if this change is really needed here. All execution path call ```ensureSaved``` before, so normally it would be ```isOffline```. Should we leave this check as a safeguard?